src/connectors/toggle-refinement/__tests__/connectToggleRefinement-test.js

```diff
@@ -3,9 +3,42 @@ import jsHelper, {
   SearchParameters,
 } from 'algoliasearch-helper';
 import connectToggleRefinement from '../connectToggleRefinement';
+import {
+  createInitOptions,
+  createRenderOptions,
+} from '../../../../test/mock/createWidget';
+import { createSearchClient } from '../../../../test/mock/createSearchClient';
+import { createSingleSearchResponse } from '../../../../test/mock/createAPIResponse';
 import { createInstantSearch } from '../../../../test/mock/createInstantSearch';
 
 describe('connectToggleRefinement', () => {
+  const createInitializedWidget = () => {
```

Comment:
@eunjae-lee I've moved the `createInitializedWidget` for `insight` function to the top level because I needed it in the `renderState` and `getWidgetRenderState` tests and updated the `helper` with `createSearchClient`